db/schema.rb

```diff
@@ -48,7 +48,6 @@
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
     t.string "point"
-    t.integer "ordering"
```

Comment:
This change was automatically generated after I ran `bundle exec rake db:migrate`

Reply:
Yea, that column was removed, but haven't been removed from schema, thanks!